### PR TITLE
Migrate to v1 of apiextensions for cni yaml

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #650

### CODE REVIEW

**Review by jaxxstorm — APPROVED** [review]

**Comment by pabardina** [comment]
Really need that one to deploy an EKS 1.22 cluster 👍

**Review by viveklak — APPROVED** [review]

File: nodejs/eks/cni/aws-k8s-cni.yaml [hunk 1]
```diff
@@ -185,7 +185,7 @@ spec:
           name: run-dir
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: eniconfigs.crd.k8s.amazonaws.com
```
Comment (viveklak) [inline nodejs/eks/cni/aws-k8s-cni.yaml:191]: The test failures suggest you need to define a schema here.. See an example here: https://github.com/argoproj/argo-helm/pull/605/files

**Comment by necaris** [comment]
Does #680 supersede this?

File: nodejs/eks/cni/aws-k8s-cni.yaml (hunk 1 above)
Comment (davidroth) [reply nodejs/eks/cni/aws-k8s-cni.yaml:191]: See https://github.com/pulumi/pulumi-eks/pull/683 which includes the latest config from upstream including missing schema from this.

**Comment by viveklak** [comment]
Closed in favor of #693

**Pull request closed without merging**